Makefile.defs

```diff
@@ -165,22 +167,18 @@ else ifeq ($(TARGET), appimage)
 	export EMULATE_READER=1
 	export SDL=1
 	export APPIMAGE=1
-else ifeq ($(TARGET), debian)
-	export SDL=1
-	export DEBIAN=1
-	export EMULATE_READER=1
-else ifeq ($(TARGET), debian-armel)
-	CHOST?=arm-linux-gnueabi
+else ifeq ($(TARGET), linux)
 	export SDL=1
-	export DEBIAN_CROSS=1
-else ifeq ($(TARGET), debian-armhf)
-	CHOST?=arm-linux-gnueabihf
-	export SDL=1
-	export DEBIAN_CROSS=1
-else ifeq ($(TARGET), debian-arm64)
-	CHOST?=aarch64-linux-gnu
-	export SDL=1
-	export DEBIAN_CROSS=1
+	ifeq ($(LINUX_ARCH), arm)
+		CHOST?=arm-linux-gnueabihf
+		export LINUX_CROSS=1
+	else ifeq ($(LINUX_ARCH), arm64)
+		CHOST?=aarch64-linux-gnu
+		export LINUX_CROSS=1
+	else
+		export LINUX=1
+		export EMULATE_READER=1
+	endif
```

Comment:
`DEBIAN_CROSS` was never used anywhere, and neither is `LINUX_CROSS`, so I'd would get rid of it.

Additionally, I would not export `LINUX`.

Reply:
Perhaps it once was in https://github.com/koreader/koreader/pull/9351? But it doesn't seem to be now in any case, so +1

Reply:
:+1: to remove the `CROSS` thingies.

The `LINUX` thingy is used in front to differenciate between "releases" and emulator builds.

Reply:
Yes, but there's still no need to export to the environment for it to be available in the front `Makefile` (AFAIK the only user).

Reply:
ok, I didn't know. Based on that I guess we don't need to export `MACOS` either.

Reply:
You still need to set it, just not with the `export` keyword (which adds the variable to the environment):

```make
	ifeq ($(LINUX_ARCH), arm)
		CHOST?=arm-linux-gnueabihf
	else ifeq ($(LINUX_ARCH), arm64)
		CHOST?=aarch64-linux-gnu
	else
		LINUX=1
		export EMULATE_READER=1
	endif

```

Reply:
> You still need to set it, just not with the `export` keyword (which adds the variable to the environment):
> 
> ```makefile
> 	ifeq ($(LINUX_ARCH), arm)
> 		CHOST?=arm-linux-gnueabihf
> 	else ifeq ($(LINUX_ARCH), arm64)
> 		CHOST?=aarch64-linux-gnu
> 	else
> 		LINUX=1
> 		export EMULATE_READER=1
> 	endif
> ```

lol :smile: 

Now everything makes sense again :)